### DISCUSSION TITLE
Why does my compilation always fail? Prompt me to migrate?

### DISCUSSION THREAD

**Discussion by bbhxwl** [prisma/prisma#24617] [Q&A]
I used nuxt3 for automatic compilation, but after running npm run build, it couldn't compile properly.

<img width="756" alt="image" src="https://github.com/prisma/prisma/assets/26525599/5529fcf3-05d2-4a96-b66f-4d2eb9720455">

**Comment by ludralph** [comment]
Hi @bbhxwl 👋 

Can you please share the complete error log and the relevant script you ran to build the project?

**Reply by bbhxwl** [reply]
https://www.npmjs.com/package/vite-plugin-cesium
I have found a problem, as long as it is used together with this, it cannot be compiled.

**Reply by bbhxwl** [reply]
> Hi @bbhxwl 👋
> 
> Can you please share the complete error log and the relevant script you ran to build the project?

https://github.com/bbhxwl/testPrisma

**Reply by bbhxwl** [reply]
![image](https://github.com/prisma/prisma/assets/26525599/74816efb-8403-42e2-a762-8ccb51ef4675)
Prisma seems to automatically delete my module. After the compilation fails, I need to perform a Yarn Install to function properly.

**Reply by ludralph** [reply]
Hi @bbhxwl 

> Prisma seems to automatically delete my module. After the compilation fails, I need to perform a Yarn Install to function properly.

Can you please give details on how this happens so we could understand the issue?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
